Modules/Module_Import/Import/xASL_imp_DetermineStructureFromSourcedata.m

```diff
@@ -83,19 +91,19 @@
     % Print overview
     xASL_imp_PrintOverview(x);
 
-
 end
 
-
+% -----------------------------------------------------------------
 %% Print overview
+% -----------------------------------------------------------------
 function xASL_imp_PrintOverview(x)
 
     if isfield(x,'importOverview')
         % Get all main fields
         overviewFields = fieldnames(x.importOverview);
         for iField=1:numel(overviewFields)
             thisSubject = overviewFields{iField};
-            if ~isempty(regexpi(thisSubject, 'subject_'))
+            if ~isempty(regexpi(thisSubject, 'subject_', 'once'))
```

Comment:
Sorry. Matlab shows this for me as a potential issue in the editor. So it's nice to have a clean list without issue. But I agree that this can be annoying, so I'll stop doing it from now on ;)